src/security/ddwaf_req.cpp

```diff
@@ -238,6 +248,60 @@ std::optional<BlockSpecification> resolve_block_spec(
 
 namespace datadog::nginx::security {
 
+namespace {
+constexpr int kMaxPlainSchemaAllowed = 260;
+constexpr int kMaxSchemaSize = 25000;
+
+template <typename Func>
+void handle_attribute(ngx_log_t &log, const ddwaf_obj &obj, Func &&f) {
+  ngx_str_t key = to_ngx_str(obj.key());
+
+  if (!obj.key().starts_with("_dd.appsec.s."sv)) {
```

Comment:
Why do you ignore other attributes, I understand you don't want to compress / encode them but at least they should be serialised as tags / metrics.